src/combinatorics/set_family/lym.lean

```diff
@@ -0,0 +1,272 @@
+/-
+Copyright (c) 2022 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta, Alena Gusakov
```

Comment:
@YaelDillies I think it's fair to add yourself to this list now!

Reply:
If you say so! 😁